cpython-unix/build.py

```diff
@@ -121,7 +121,7 @@ def add_target_env(env, build_platform, target_triple, build_env):
             .replace("x86_64_v3-", "x86_64-")
             .replace("x86_64_v4-", "x86_64-")
             # TODO should the musl target be normalized?
-            .replace("-unknown-linux-musl", "-unknown-linux-gnu")
+            # .replace("-unknown-linux-musl", "-unknown-linux-gnu")
```

Comment:
The answer to this todo seems to be: no it should not generally be normalized like this because it causes problems. But then there are a couple specific builds where it _does_  need to be normalized because there isn't support for `-musl` targets?